net/libfab/libfab.c

```diff
@@ -74,11 +76,17 @@ struct transfer_ma {
 	*/
 };
 
+#define FAB_MR_ACCESS  (FI_READ | FI_WRITE | FI_RECV | FI_SEND | \
+			FI_REMOTE_READ | FI_REMOTE_WRITE)
+#define FAB_MR_OFFSET   0
+#define FAB_MR_FLAG     0
+#define FAB_MR_KEY      0XABCD
```

Comment:
Use 'enum' instead of 'define' is possible.

Reply:
will use enum